tests/int/index.js

```diff
@@ -121,7 +121,8 @@ describe('couch2pg', function() {
     var importer = require('../../lib/importer')(
       pgdb, couchdb,
       COUCH2PG_DOC_LIMIT,
-      COUCH2PG_CHANGES_LIMIT
+      COUCH2PG_CHANGES_LIMIT,
+      'default-medic'
```

Comment:
This doesn't matter sooooo much cause it's a test, but FWIW `couch2pg` by itself should not have anything medic specific in it, that is what `medic-couch2pg` (the app that actually gets deployed, that wraps this library) is for.